cmd/daemon/controller.go

```diff
@@ -35,17 +37,35 @@ var (
 			"present within the cluster",
 		Run: runControllerCmd,
 	}
-	kubeconfig string
+	kubeconfig     string
+	windowsService bool
+	logDir         string
 )
 
 func init() {
 	rootCmd.AddCommand(controllerCmd)
 	controllerCmd.PersistentFlags().StringVar(&kubeconfig, "kubeconfig", "", "Path to kubeconfig")
+	controllerCmd.PersistentFlags().StringVar(&logDir, "log-dir", "", "Directory to write logs to")
+	controllerCmd.PersistentFlags().BoolVar(&windowsService, "windows-service", false,
+		"Enables running as a Windows service")
 }
 
 func runControllerCmd(cmd *cobra.Command, args []string) {
+	if logDir != "" {
+		var fs flag.FlagSet
+		klog.InitFlags(&fs)
+		fs.Set("logtostderr", "false")
+		fs.Set("log_dir", logDir)
```

Comment:
Is there a default `log_dir` value, in case no `logDir` is provided? If so, you should mention it in the command's [description](https://github.com/openshift/windows-machine-config-operator/pull/1081/files#diff-dcdb946703bcd804e30b1506a5a49888d8945bef67c7f354d1a52559b90bc381R48).

Reply:
+1